views/networking/firewall/index.erb

```diff
@@ -0,0 +1,57 @@
+<% @page_title = "Firewalls" %>
+  <%== render("networking/tabbar") %>
+    <% if @firewalls.count > 0 %>
```

Comment:
The head indentation is inconsistent between "firewall/index.erb" and "private_subnet/index.erb"